app/code/Magento/InventoryApi/Api/GetAvailableSourceItemsBySkusAndSortedSourceInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryApi\Api;
+
+/**
+ * Retrieve source items for a defined set of skus and sorted source codes
+ *
+ * Useful for determining availability and source selection
+ *
+ * @api
+ */
+interface GetAvailableSourceItemsBySkusAndSortedSourceInterface
```

Comment:
I'd suggest moving this interface away from `InventoryApi`, and even avoiding `@api` annotation.

Even the name reveals that this is something created specifically for the SSA, therefore creating a leaky abstraction.

Reply:
100% agree. I reworked it to avoid modules DB dependencies.